dpnp/tests/helper.py

```diff
@@ -197,13 +203,15 @@ def generate_random_numpy_array(
 
     """
 
-    numpy.random.seed(seed_value)
+    numpy.random.seed(seed_value) if seed_value else numpy.random.seed(42)
```

Comment:
Would it be more readable?
```suggestion
    if seed_value is None:
        seed_value = 42
    numpy.random.seed(seed_value)
```
or probably should we set default value to `42`?